src/tasks/daily/daily_battle_mixin.py

```
@@ -399,41 +431,77 @@ def _gather_retry_navigate(self, stage_name, category_name):
+        # 如果是放弃领奖，那么还需要点击确认
+        if is_extra_mode and self.wait_click_ocr(match=re.compile("确认"), box=self.box.bottom_right, time_out=5, recheck_time=1, alt=True):
+            self.log_info("已放弃未领取的奖励")
         return True
 
-    def battle_recycle(self, left_ticket, stage_name, category_name, enter_str, no_battle=False, challenge_check=False):
+    def to_restart(self, is_extra_mode, enter_box):
+        if is_extra_mode:
+            # 放弃领取奖励后需要重新点击激发按钮
+            result = self.wait_ocr(match=re.compile("激发"), box=self.box.bottom_right, time_out=5)
+            if not result:
+                self.log_info("未找到『激发』按钮，无法继续进行额外刷取")
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)